collect_app/build.gradle

```diff
@@ -139,10 +139,9 @@ allprojects {
 
 dependencies {
     compile fileTree(dir: 'libs', include: '*.jar')
-    compile group: 'com.google.android.gms', name: 'play-services-analytics', version: '10.0.1'
-    compile group: 'com.google.android.gms', name: 'play-services-maps', version: '10.0.1'
-    compile group: 'com.google.android.gms', name: 'play-services-auth', version: '10.0.1'
-    compile group: 'com.android.support', name: 'multidex', version: '1.0.1'
+    compile group: 'com.google.android.gms', name: 'play-services-analytics', version: '10.2.0'
```

Comment:
We do not want to update Google Play Services in this next release. That would force certain devices to download a big file for the upgrade.